AlphaWallet/AppDelegate.swift

```diff
@@ -123,12 +124,37 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
                 if let endpointArnForSNS = createEndpointResponse.endpointArn {
                     print("endpointArn: \(endpointArnForSNS)")
                     UserDefaults.standard.set(endpointArnForSNS, forKey: "endpointArnForSNS")
+                    //every user should subscribe to the security topic
+                    self.subscribeToTopicSNS(token: token, topicEndpoint: self.SNSSecurityTopicEndpoint)
+                    if let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+                        //TODO subscribe to version topic when created
```

Comment:
We can perform filtering for version numbers when the app receive a silent notification (though it's not perfect), but it should help to reduce some housekeeping work for creating and maintaining a topic for each version.

Worth a refactor to reduce code added to the AppDelegate though. Maybe later.